Xamarin.Forms.Core/Xamarin.Forms.Core.csproj

```diff
@@ -22,17 +22,8 @@
 	</ItemGroup>
 	<Import Project="..\Xamarin.Flex\Xamarin.Flex.projitems" Label="Shared" Condition="Exists('..\Xamarin.Flex\Xamarin.Flex.projitems')" />
 	<UsingTask TaskName="XFCorePostProcessor.Tasks.FixXFCoreAssembly" AssemblyFile="..\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll" />
-	<Target
-		Condition="$(DesignTimeBuild) != true AND $(BuildingProject) == true"
-		AfterTargets="AfterCompile"
-		Name="XFCorePostProcessor"
-		Inputs="$(IntermediateOutputPath)$(TargetFileName)"
-		Outputs="$(IntermediateOutputPath)XFCorePostProcessor.stamp">
-	    <Touch
-		    Files="$(IntermediateOutputPath)XFCorePostProcessor.stamp"
-		    AlwaysCreate="True" />
-	    <FixXFCoreAssembly
-		    Assembly="$(IntermediateOutputPath)$(TargetFileName)"
-		    ReferencePath="@(ReferencePath)" />
+	<Target Condition="$(DesignTimeBuild) != true AND $(BuildingProject) == true" AfterTargets="AfterCompile" Name="XFCorePostProcessor" Inputs="$(IntermediateOutputPath)$(TargetFileName)" Outputs="$(IntermediateOutputPath)XFCorePostProcessor.stamp">
```

Comment:
I like the way they are. don't change formatting if you do not change the actual content

Reply:
I don't think we're going to get around this change. The IDE is making this change so it's going to cause a lot of headaches for everyone to selectively not commit this